src/to-do-List/typings/span-task-properties.ts

```diff
@@ -0,0 +1,6 @@
+export interface SpanProperties {
```

Comment:
This properties are never reused so it would be better to move them to the component's file